### PR TITLE
AAE-2342 Change internal port place

### PR DESCRIPTION
**Pull request by miguelruizdev**
Addressing this issue: https://issues.alfresco.com/jira/browse/AAE-2342

### CODE REVIEW

File: helm/alfresco-adf-app/values.yaml [hunk 1]
```diff
@@ -40,10 +40,10 @@ image:
   repository: quay.io/alfresco/alfresco-adf-app
   tag: latest
   pullPolicy: IfNotPresent
-  internalPort: null
 service:
   type: ClusterIP
   port: 80
+  internalPort: 8080
```
Comment (ffazzini) [inline helm/alfresco-adf-app/values.yaml:46]: can you check in the infrastructure chart if modeling-app / admin-app need to be updated in order to use the new field?

Comment (miguelruizdev) [reply helm/alfresco-adf-app/values.yaml:46]: true, we need to refactor that for the apps: https://github.com/Alfresco/alfresco-process-infrastructure-deployment/blob/develop/helm/alfresco-process-infrastructure/values.yaml#L92

Comment (ffazzini) [reply helm/alfresco-adf-app/values.yaml:46]: yes maybe you can just remove that field from the infrastructure as default will already be 8080

Comment (miguelruizdev) [reply helm/alfresco-adf-app/values.yaml:46]: here: https://github.com/Alfresco/alfresco-process-infrastructure-deployment/pull/16

**Pull request merged**